index.bs

```diff
@@ -192,30 +192,35 @@ the user agent MUST run the following steps:
 
 1. If <a>Picture-in-Picture support</a> is `false`, throw a
     {{NotSupportedError}} and abort these steps.
-2. If the document is not <a>allowed to use</a> the <a>policy-controlled feature</a>
+2. If the document is not [=/allowed to use=] the <a>policy-controlled feature</a>
     named `"picture-in-picture"`, throw a {{SecurityError}} and abort these
     steps.
-3. If |video|'s {{readyState}} attribute is {{HAVE_NOTHING}}, throw a
+3. If the document is not [=/allowed to use=] the <a>policy-controlled feature</a>
+    named `"automatic-picture-in-picture"` and the
```

Comment:
Where did the enum come from? Was it from the document? 

I see there is some connection to media session, but it's not how this is checked